tests/conftest.py

```diff
@@ -386,7 +386,9 @@ async def autostarted_agent(server, environment):
     await env.set(data.AUTOSTART_AGENT_MAP, {"agent1": ""})
     await env.set(data.AUTO_DEPLOY, True)
     await env.set(data.PUSH_ON_AUTO_DEPLOY, True)
-    await env.set(data.AUTOSTART_AGENT_DEPLOY_SPLAY_TIME, 0)
+    # disable deploy and repair intervals
+    await env.set(data.AUTOSTART_AGENT_DEPLOY_INTERVAL, 0)
+    await env.set(data.AUTOSTART_AGENT_REPAIR_INTERVAL, 0)
```

Comment:
No failing tests caused by this change on core. It seems to me (after discussion with @arnaudsjs) like it was a mistake previously, because it allowed for multiple deployment triggers. But because of my limited experience with this I'd like the reviewer to review this part critically.

Reply:
Looks good to me, it at least makes it more deterministic.